lib/gs-utils.c

```diff
@@ -1041,4 +1041,20 @@ gs_utils_get_memory_total (void)
 	return si.totalram / MB_IN_BYTES / si.mem_unit;
 }
 
+/**
+ * gs_utils_set_install_timestamp:
+ *
+ * Sets the value of install-timestamp which represents the last time
+ * we had an update or upgrade.
+ *
+ **/
+void
+gs_utils_set_install_timestamp (GSettings *settings)
```

Comment:
Nitpick: I'd prefer `.._update_install_timestamp` because set usually takes an argument, and I don't know if we will need to add that argument in the future.

Reply:
hmm, you make a good point. How about we make this change while we update `install-timestamp` to `update-timestamp`. Does it co-relate? 

I should probably update the title of PR too. :laughing:

Reply:
I don't see how it correlates. In your PR you're also updating the timestamp on every update, not on every install, thus the incoherence between the name and what it does is independently from how we set it on updates.